templates/post-preview.php

```diff
@@ -14,8 +14,12 @@
 		404
 	);
 }
-$user   = \Activitypub\Collection\Users::get_by_id( $post->post_author );
-$object = $transformer->to_object();
+
+$object        = $transformer->to_object();
+$attributed_to = $object->get_attributed_to();
+$user_uri      = is_array( $attributed_to ) ? reset( $attributed_to )[0] : $attributed_to;
+$user          = \Activitypub\Collection\Users::get_by_resource( $user_uri );
```

Comment:
what about making `$transformer->get_actor_object();` accessible? this feels like we do the same over and over!?!

Reply:
I favour the idea and would also suggest to investigate whether the whole logic of "which" actor is `owing` a post, and who is announcing or creating/updating could be more in one place. 
FYI: My background is that I wonder if it could be possible in the feature also create new actors for event organizers, but currently this seems like a very distant goal which may also never be achieved due it's complexity, both here for us devs and for the users.

Reply:
totally! I will start working on the outbox feature this week and try to do it as open as possible and to start painting the overall idea. I think this could make this a lot easier. At least I will try to keep your feedback about that in mind!

Reply:
🤔 The more I look into the code, the more I agree with your! I think we have to refactor this a bit within the Outbox task!

And to simplify the transformer, so that it will always return the correct user regardless of the post-author!

Reply:
@Menrath I did some changes that should work for now. Let me know what you think!